npm/pkg/dataplane/policies/policymanager.go

```diff
@@ -92,6 +94,11 @@ func (pMgr *PolicyManager) Bootup(epIDs []string) error {
 		// update Prometheus metrics on success
 		metrics.IncNumACLRulesBy(numLinuxBaseACLRules)
 	}
+
+	if util.IsWindowsDP() && pMgr.NodeIP == "" {
+		return npmerrors.Errorf(npmerrors.BootupPolicyMgr, false, "policy manager must have a configured nodeIP in Windows")
```

Comment:
This return error might be fatal, but should be okay for time being. We will need to make our getnodeip logic robust in case we see any failures. 

Can put up a alert on this error message from telemetry.